2023/YoctoSummit-2023.11/uefi-sb-with-yocto.md

```diff
@@ -0,0 +1,31 @@
+# Title
+
+Dealing with UEFI Secure Boot support using Yocto Project
+
+# Abstract
+
+UEFI Secure Boot, a standard within the UEFI framework, enhances embedded
+platform security by verifying trusted keys for loaded images in the UEFI BIOS.
+However, integrating it into a project involves challenges such as ensuring UEFI
```

Comment:
(...) ensuring the level of UEFI specification compliance that UEFI BIOS implementation on a given hardware target provides. Not knowing what is supported may lead to poor choices during integration, and not all UEFI BIOS implementations comply similarly with the specification. Those can also comply with various versions of the specification, which may lead to differences in what and how UEFI Secure Boot should be supported.

@TomaszAIR This is a very long rant, and you should not copy that, but I hope you get my point

Reply:
@pietrushnic I just added the first part of that https://github.com/3mdeb/conferences/pull/9/commits/396893ac0961f582b199b120334541410d29657b

Definitely I will talk more about it in the presentation.